sujeong/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "myapp",
+  "version": "1.0.0",
+  "main": "app.js",
+  "scripts": {
+    "start": "nodemon app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "devDependencies": {
+    "express": "^4.18.2",
+    "nodemon": "^2.0.20"
+  },
+  "dependencies": {
+    "cors": "^2.8.5",
+    "dbmate": "^1.0.3",
+    "dotenv": "^16.0.3",
+    "morgan": "^1.10.0",
+    "mysql": "^2.18.1",
+    "typeorm": "^0.3.10"
+  }
+}
```

express의 경우 해당 프로젝트를 사용하기 위해서 필수적으로 존재해야하는 패키지입니다. 그렇기에 devdependencies가 아닌 dependencies에 관리되어야 합니다 !

```suggestion
  "devDependencies": {
    "nodemon": "^2.0.20"
  },
  "dependencies": {
    "cors": "^2.8.5",
    "dbmate": "^1.0.3",
    "dotenv": "^16.0.3",
    "morgan": "^1.10.0",
    "mysql": "^2.18.1",
    "typeorm": "^0.3.10"
    "express": "^4.18.2",
  }
}
```